codegen/src/main/java/software/amazon/awssdk/codegen/poet/batchmanager/BaseBatchManagerClassSpec.java

```diff
@@ -144,9 +144,7 @@ private MethodSpec constructor() {
     }
 
     private MethodSpec internalTestConstructor() {
-        MethodSpec.Builder builder = MethodSpec.constructorBuilder()
-                                               .addModifiers(PUBLIC)
-                                               .addAnnotation(SdkInternalApi.class);
+        MethodSpec.Builder builder = MethodSpec.constructorBuilder();
```

Comment:
nit, can we add `SdkInternalApi` annotation?

Reply:
Added `SdkInternalApi`